src/components/TokenClaimDialog.tsx

```diff
@@ -188,19 +233,42 @@ export default function TokenClaimDialog({
               </Trans>
             </Typography>
           )}
-          {allPoolsWithRewards.map((pool, i, arr) => (
-            <React.Fragment key={pool.poolName}>
-              <ClaimListItem
-                title={pool.poolName}
-                amount={rewardBalances[pool.poolName] || Zero}
-                claimCallback={() => claimPoolReward(pool)}
-                status={
-                  claimsStatuses["allPools"] || claimsStatuses[pool.poolName]
-                }
-              />
-              {i < arr.length - 1 && <Divider key={i} />}
-            </React.Fragment>
-          ))}
+          {!isVeSdlEnabled
```

Comment:
In light of the conversation we had on Friday about the mechanics of the token launch, this will need to be a bit more nuanced. 
The condition under which we show gauges is `IS_VESDL_LIVE` AND `chainId === ChainId.MAINNET`. We should probably create a function or hook to house this logic in case it changes (eg we also launch gauges on arbitrum)

cc @penandlim  to confirm that we're still planning to use minichef on non-mainnet to serve rewards which we voted on via minichef

Reply:
I still dont think its a good idea to have intention to support cross chain gauge right off the bat if they are not ready. Although we can use minichef on other chains, it would require multiple time sensitive coordination with our current multisig.

But if we reallllly want to do this, we can.